modules/graphql_views/src/Plugin/GraphQL/Fields/View.php

```diff
@@ -67,6 +67,17 @@ public function resolveValues($value, array $args, ResolveInfo $info) {
       $executable = $view->getExecutable();
       $executable->setDisplay($definition['display']);
 
+      // Set view arguments (contextual filters) if required.
+      /* @see \Drupal\graphql_views\Plugin\Deriver\ViewDeriver::getContextualSets() */
+      foreach ($definition['contextual_sets'] as $contextual_set) {
+        if (isset($contextual_set['argument_entity_class']) && is_a($value, $contextual_set['argument_entity_class'])) {
+          $executable->setArguments([
+            $contextual_set['argument'] => $value->id(),
+          ]);
+          break;
```

Comment:
Do we want to break here? Or pass the contextual argument everywhere it fits?

I just created a "related articles" field based on terms relations. In this case I would use the current node twice. Once for the relation and once to exclude itself from the result.

Might feel "magic" ... but could be useful.